src/main/java/studos/logic/I18N.java

```diff
@@ -61,8 +61,17 @@ private static boolean languageReadFromConfig() {
      * @return List of Locale objects.
      */
     public static List<Locale> getSupportedLocales() {
-        return new ArrayList<>(Arrays.asList(Locale.forLanguageTag("en"),
-         Locale.forLanguageTag("pl")));
+        return new ArrayList<>(Arrays.asList(Locale.forLanguageTag("pl-PL"),
+        Locale.forLanguageTag("en-US")));
+    }
+
+    /**
+     * UserConfigReader which correct locale.
+     * @return Locale
+     */
+    public static Locale getChangeLocale() {
```

Comment:
You mean "getChangedLocale"? This is checking language change method or correcting language form config file method?

If it's a method that takes language from config file then you should name it something like "getUserConfigLanguage".